src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1107,6 +1107,17 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 #endif
 	}
 	case SN_ConvertToInt32: 
+#ifdef TARGET_AMD64
```

Comment:
This looks a bit weird, this code should be in the original case block inside an if (id == SN_ConvertToInt32).